### PR TITLE
bugzilla: add a daily refresh to catch BZ state changes

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @eparis @derekwaynecarr

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -144,6 +144,49 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 1 * * *
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml:149]: This is 1:00am UTC or 3:00am CET or 6:00pm PST

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -144,6 +144,49 @@ periodics:
...
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-daily-bugzilla-refresh
+  spec:
+    containers:
+    - args:
+      - |-
+        --query=is:pr
+        state:open
+        label:bugzilla/invalid-bug
```
Comment (eparis) [inline ci-operator/jobs/infra-periodics.yaml:160]: Adding label:lgtm might reduce the number even further. I'm not sure all repos used approved, so maybe not that....

**Comment by stevekuznetsov** [comment]
Preexisting one is for humans, read the comment it posts :)

**Pull request merged**